CHANGELOG.md

```diff
@@ -55,6 +55,7 @@ All notable changes to this project will be documented in this file.
 -   #1930 : Preserve ordering of import targets.
 -   #1951 : Fix return type for class whose argument cannot be wrapped.
 -   #1892 : Fix implementation of list function when an iterable is passed as parameter.
+-   #1924 : Fix internal error arising in Duplicate or list comprehensions.
```

Comment:
#1924 is the number of the issue not the PR.

Reply:
Yes. The PR is only referenced here when there is no issue. The issue is the thing that contains the detailed user-friendly description of what has been fixed

Reply:
E.g. the 3 above:
- https://github.com/pyccel/pyccel/issues/1892
- https://github.com/pyccel/pyccel/issues/1951
- https://github.com/pyccel/pyccel/pull/1930 (PR not linked to an issue)